src/themes/theme.tsx

```diff
@@ -0,0 +1,14 @@
+import { DefaultTheme, FlattenSimpleInterpolation } from "styled-components";
```

Comment:
Nice and clean file! Well done! I really like that we have a base for possible future theme enhancements!